docs/plugins/filters/ruby.asciidoc

```diff
@@ -50,8 +141,11 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 [cols="<,<,<",options="header",]
 |=======================================================================
 |Setting |Input type|Required
-| <<plugins-{type}s-{plugin}-code>> |<<string,string>>|Yes
+| <<plugins-{type}s-{plugin}-code>> |<<string,string>>|No
 | <<plugins-{type}s-{plugin}-init>> |<<string,string>>|No
+| <<plugins-{type}s-{plugin}-path>> |<<string,string>>|No
+| <<plugins-{type}s-{plugin}-script_params>> |<<hash,hash>>,{}|No
+| <<plugins-{type}s-{plugin}-tag_on_exception>> |<<string,string>>,_rubyexception|No
```

Comment:
This ID causes a doc build error because the description of the tag_on_exception config option doesn't exit.

Reply:
Maybe this should be tag_on_failure?